include/sot/core/robot-utils.hh

```diff
@@ -217,28 +259,74 @@ public:
       If parameter_name already exists the value is overwritten.
       If not it is inserted.
    */
+  template < typename Type>
   void set_parameter(const std::string &parameter_name,
-                     const std::string &parameter_value);
+                     const Type &parameter_value)
+  {
+    try
+    {
+      typedef boost::property_tree::ptree::path_type path;
+      path apath(parameter_name,'/');
+      property_tree_.put<Type>(apath,parameter_value);
+    }
+    catch(const boost::property_tree::ptree_error &e)
+    {
+      std::ostringstream oss;
+      oss << "Robot utils: parameter path is invalid " << '\n'
+          << " for set_parameter("
+          << parameter_name << ")\n"
+          << e.what() << std::endl;
+      sendMsg(oss.str(),
+              MSG_TYPE_ERROR);
```

Comment:
```suggestion
      DYNAMIC_GRAPH_ENTITY_ERROR(*this) << "Robot utils: parameter path is invalid " << '\n'
          << " for set_parameter("
          << parameter_name << ")\n"
          << e.what();
```

Reply:
I was too fast. RobotUtils class is not an entity.

Reply:
Here, it is not an entity, so you can use `dgRTLOG() << ...`